apps/website/src/components/LeftSidebarContent.astro

```diff
@@ -88,11 +90,35 @@ sections.set(
     }
   }
 
-  .nav-group-title {
+  .nav-group-title a {
     color: var(--color-subtext);
     font-size: var(--type-0);
     font-weight: 600;
-    padding-inline: var(--space-4);
+    padding: 0.3rem var(--space-4);
+    margin-bottom: var(--space-2);
+    text-decoration: none;
+    text-transform: uppercase;
+    display: block;
+
+    &:where(:hover, :focus) {
+      background-color: var(--color-active);
+      color: var(--color-text);
+    }
+
+    @media (max-width: 50em) {
+      color: var(--color-text);
+      font-size: var(--type-3);
+      padding-inline: var(--space-6);
+      margin-bottom: var(--space-3);
+    }
+  }
+
+  .nav-group-subtitle {
```

Comment:
the styling is a bit confusing now and it's not obvious what is clickable vs what isn't.
![](https://user-images.githubusercontent.com/9084735/206533646-828e07f8-1113-4b0b-a9c7-0614e0940476.png)

Reply:
Should be good now. Used --color-line-2 for muted look. 
![image](https://user-images.githubusercontent.com/83585998/207624366-5e560d14-c434-4885-9cfc-193a706f7d5c.png)

Reply:
alignment looks better now but `--color-line-2` has bad contrast because it's not really meant to be used as for text. might want to add a new muted color that meets 4.5 contrast score

Reply:
![image](https://user-images.githubusercontent.com/83585998/207847319-8edb3220-8364-4a03-a08f-9effae464090.png)

Ratio 4:63